.travis.yml

```diff
@@ -33,7 +33,7 @@ matrix:
     - dist: bionic
       python: 3.9-dev
       env:
-        - JOB="3.9-dev" PATTERN="(not slow and not network and not clipboard)"
+        - JOB="3.9-dev" PATTERN="(not slow and not network and not clipboard)" TEST_ARGS="-Werror"
```

Comment:
This gets passed to pytest right? I think the flag there is just `-W`

https://docs.pytest.org/en/latest/warnings.html

Reply:
Or could even do in the setup.cfg

Reply:
Trying it out in setup.cfg. The primary issue I see is that we don't necessarily want warnings from optional dependencies to cause errors. For now I've limited to just our required dependencies.